README.rst

```diff
@@ -1,5 +1,6 @@
 BigML PHP Bindings
 =====================
+
 In this repository you'll find an open source PHP client that gives you a simple binding to interact with `BigML <https://bigml.com>`_.
```

Comment:
I think the following:

In this repository you'll find an open source PHP library that gives you a simple way to interact...

works better

Reply:
Excellent, I'll change the language.